doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,288 @@
+
+      virt-install \
+      --connect qemu:///system \
+      --name ACRNSOS \
+      --machine q35 \
+      --cpu host-passthrough,+invtsc \
+      --ram 4096 \
+      --disk path=/var/lib/libvirt/images/acrnsos.img,size=32 \
+      --vcpus 4 \
+      --virt-type kvm \
+      --os-type linux \
+      --os-variant ubuntu18.04 \
+      --graphics none \
+      --clock offset=utc,tsc_present=yes,kvmclock_present=no \
+      --qemu-commandline="-machine kernel-irqchip=split -device intel-iommu,intremap=on,caching-mode=on,aw-bits=48" \
+      --location 'http://archive.ubuntu.com/ubuntu/dists/bionic/main/installer-amd64/' \
+      --extra-args "console=tty0 console=ttyS0,115200n8"
+
+2. Walk through the installation steps as prompted. Few things to note,
+
+   a. Make sure to install an OpenSSH server so that once the installation is complete, we can SSH into the system.
+
+      .. figure:: images/acrn_qemu_1.png
+         :align: center
+
+   b. We use GRUB to boot ACRN, so please install it when prompted.
+
+      .. figure:: images/acrn_qemu_2.png
+         :align: center
+
+3. To login to the Service VM guest, find the IP address of the guest to SSH. This can be done via the
+   virsh command as shown below,
+
+   .. figure:: images/acrn_qemu_3.png
+      :align: center
+
+4. Once ACRN hypervisor is enabled, the above virsh command might not display the IP. So enable Serial console by,
+
+   .. code-block:: none
+
+      sudo systemctl enable serial-getty@ttyS0.service
+      sudo systemctl start serial-getty@ttyS0.service
+
+5. Enable GRUB menu to choose between Ubuntu vs ACRN hypervisor. Modify :file:`/etc/default/grub` and edit below entries,
+
+      .. code-block:: none
+
+         GRUB_TIMEOUT_STYLE=menu
+         GRUB_TIMEOUT=5
+         GRUB_CMDLINE_LINUX_DEFAULT=""
+         GRUB_GFXMODE=text
+
+6. Update GRUB changes by ``sudo update-grub``
+
+7. Once the above steps are done, Service VM guest can also be launched using, ``virsh start ACRNSOS --console``. Make sure to use the domain name
+   you used while creating the VM instead of ``ACRNSOS``.
+
+This concludes setting up of Service VM and preparing it to boot ACRN hypervisor.
+
+.. _install_acrn_hypervisor:
+
+Install ACRN Hypervisor
+***********************
+
+1. Please clone ACRN repo with ``tag: acrn-2020w19.5-140000p`` or latest master. Below steps show our tested version,
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/
+      $ git fetch --all --tags --prune
+      $ git checkout tags/acrn-2020w19.5-140000p -b acrn_on_qemu
+
+2. Use the following command to build ACRN for QEMU,
+
+   .. code-block:: none
+
+      make all BOARD_FILE=./misc/acrn-config/xmls/board-xmls/qemu.xml  SCENARIO_FILE=./misc/acrn-config/xmls/config-xmls/qemu/sdc.xml
+
+ For more details, refer to :ref:`getting-started-building`.
+
+3. Copy ``acrn.32.out`` from ``build/hypervisor`` to Service VM guest ``/boot/`` directory.
```

Comment:
Use a code-block with commands instead of a sentence.

Reply:
The user can build in a different system and copy it SUT. So cannot explicitly put this as command and so decided to give it as an instruction.